### ISSUE TITLE
Most 3/23/20 entries are duplicates of 3/22/20

### ISSUE THREAD

**Issue by arik-so** [CSSEGISandData/COVID-19#1321]
The new data is mostly missing for today, but it's less obvious than it used to be because instead of empty cells, values are propagated from the previous day.

**Comment by bevanward** [comment]
@arik-so looks like a lot are empty too - if you scan down the list you see the data missing.

**Comment by jclerman** [comment]
To be clear, this issue affects both the old-style (but still in use) time-series files (`time_series_19-covid-Confirmed.csv` etc) and the `03-23-2020.csv` daily-report file.

**Comment by Shane073** [comment]
Total count on the dashboard about 2 hours ago was 375,458 confirmed cases. Latest download total confirmed cases are 336,004. Sorry, but you guys really need to fix these errors up.

**Comment by j-fu** [comment]
... indeed

**Comment by ddub03** [comment]
agreed....
is anyone tracking the number of CV19 tests by location, compared to the number returned to see which locations might be spiking next? seems like there's a lack of tests given/tests returned data

**Comment by PlentyJeff** [comment]
There's a good repository of testing data here: https://covidtracking.com/

**Comment by j-fu** [comment]
> ... indeed

;-) The new format files seem to be ok now. Thanks & stay healthy!

**Comment by arik-so** [comment]
It also appears that the US data is not included in the global time series CSVs, but the US CSVs are also not uploaded yet.

**Comment by thepost** [comment]
The US data is updating [for me](http://coronavirus.doomerfood.com/), except for **recoveries** (which totals to 0 for each row).

**Comment by alexchandel** [comment]
@CSSEGISandData Please un-corrupt the deprecated CSVs in csse_covid_19_time_series.  I'm still using them.

**Comment by maisonml** [comment]
Recovered cases data is missing for US and Canada in csse_covid_19_daily_reports/03-23-2020.csv and for UK, recovered cases overall has dropped from yesterday's count.

**Comment by thepost** [comment]
> Recovered cases data is missing for US and Canada in csse_covid_19_daily_reports/03-23-2020.csv and for UK, recovered cases overall has dropped from yesterday's count.

That's true for Canada and the US. UK checks out ok for me, recovered is at 140 vs 67 yesterday.

**Comment by LucidObscurity** [comment]
They've deprecated those files. Now you have to use the ...global... files for the latest data. They will not longer be collecting "Recovered" data also. For US state data, I guess we have to wait until the new ...us... files are ready.

**Comment by LucidObscurity** [comment]
[From Here](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series)

<b>---DEPRICATED WARNING---</b><br>
The files below will no longer be updated. With the release of the new data structure, we are updating our time series tables to reflect these changes. Please reference time_series_covid19_confirmed_global.csv and time_series_covid19_deaths_global.csv for the latest time series data. <br><br>
time_series_19-covid-Confirmed.csv	<br>
time_series_19-covid-Deaths.csv	<br>
time_series_19-covid-Recovered.csv

**Comment by alexchandel** [comment]
@LucidObscurity Deprecated does not mean "corrupted."  Regardless of deprecation status, @CSSEGISandData **must** uncorrupt the files they broke today.

**Comment by thepost** [comment]
Deprecated does mean unreliable though. @alexchandel you have more of a responsibility to update your codebase than @CSSEGISandData does to maintain deprecated endpoints.
